system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -47,7 +47,7 @@
         fill: $fill-primary;
       }
 
-      &[aria-expanded='true'] + .info-option {
+      .info-option {
```

Comment:
These two are not the same before/after. Note the `+`.

Reply:
Fixed with commit https://github.com/mozilla/activity-stream/pull/3751/commits/cb0617360bf0fecdb23191a6f0618a3f63f558ec